bcgov/mds
# MDS-4147: Add notes to each mine incident

# Main

- Ticket; https://bcmines.atlassian.net/jira/software/c/projects/MDS/boards/1?modal=detail&selectedIssue=MDS-4147
- Add "Communication bubble" to the Incidents and Investigation table
- Create new MineIncidentNotes modal to handle creation/deletion and display of notes for an incident
- Add new role(core_edit_incidents) and applied it to certain groups on Test/Prod(Health & Safety, Major Mines Permitting, Regional Mines Permitting, Compliance & Enforcement)

# How to test

- Pre-requisite: Navigate to any mine that has an incident/create a new incident
- On the incident table ensure every record has an action icon(far right) that is a chat bubble
- Click the Chat Bubble icon and be presented with this: 
![image](https://user-images.githubusercontent.com/13891487/157741705-39343d3f-ee8d-426f-b6be-93af930ca18e.png)
- Type in a new note and click "Add Comment" -> ensure the comment panel updates
- Add another note and ensure the messages are ordered from newest to oldest(top to bottom)
- Delete a note and ensure it gets removed correctly
